specification/playwrighttesting/PlaywrightTesting.Management/main.tsp

```diff
@@ -0,0 +1,175 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+//import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+using Azure.Core;
+
+@armProviderNamespace("Microsoft.AzurePlaywrightService")
+@service({
+  title: "Microsoft AzurePlaywrightService Management API",
+  version: "2023-10-01-preview",
+})
+@doc("Microsoft.AzurePlaywrightService Resource Provider management API.")
+@useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+@useDependency(Azure.Core.Versions.v1_0_Preview_2)
+namespace Microsoft.AzurePlaywrightService;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-path-segment-invalid-chars" "Existing Template"
```

Comment:
>#suppress [](http://example.com/codeflow?start=0&length=9)

Why are these suppress statements needed?

Reply:
were present in auto-generated typespec doc for provider Hub User RP. Removed them.